stacks-signer/Cargo.toml

```diff
@@ -35,11 +35,14 @@ slog = { version = "2.5.2", features = [ "max_level_trace" ] }
 slog-json = { version = "2.3.0", optional = true }
 slog-term = "2.6.0"
 stacks-common = { path = "../stacks-common" }
+stackslib = { path = "../stackslib"}
 thiserror = "1.0"
 toml = "0.5.6"
 tracing = "0.1.37"
 tracing-subscriber = { version = "0.3.17", features = ["env-filter"] }
 wsts = { git = "https://github.com/Trust-Machines/wsts", tag = "4.0.0rc2" }
+tinyhttp = "0.3.8"
+reqwest = { version = "0.11.22", features = ["blocking", "json"] }
```

Comment:
I do believe Jude has concerns about using reqwest because of unsafe code in the blocking call. We may need to hunt around for a different crate :(

Reply:
I would also be happy with allowing this through for now and making a ticket to repalce it with some alternative. I think the HTTP code throughout stacks is being pulled into a lib and maybe we could reuse that?

Reply:
> I do believe Jude has concerns about using reqwest because of unsafe code in the blocking call. We may need to hunt around for a different crate :(

I have many, many concerns about `reqwest` -- in particular, the god-awful dependencies it pulls in.

If you're only doing blocking calls, I'd recommend `ureq`.  No need to import the whole and run the whole of `tokio` just to read and write data on a socket.

> I really really really do not want to have to implement a custom http client.

Oh but it's so much fun! ;)

Reply:
@soju-drinker pulled out the ureq suggestions into a separate issue - https://github.com/stacks-network/stacks-blockchain/issues/3995

Hopefully we can get this merged asap to build off of (prevent massive PR)

Reply:
Just to be clear, this is going to need to be addressed before a merge to `master`.